File: gobsprogram.js
var readline = require('readline');

var rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

var cols = process.stdout.columns;
var line = "";

for (i = 0; (line.length + 7) <= cols; i++) {
  if (i > 0) {
    line += "  ";
  }

  line += "Penus";
}

line += "\n";

rl.question("Gob's Program:  Y/N?\n? ", function (answer) {
  if (answer.match(/Y/i)) {
    while (true) {
      process.stdout.write(line);
    }
  }

  rl.close();
});

File: gobsprogram.go
package main

import "fmt"

func main() {
	var answer string
	fmt.Print("\033[1;32m")
	fmt.Print("\033[40m")
	fmt.Println("Gob's Program:  Y/N?")
	fmt.Print("? ")
	fmt.Scanf("%s", &answer)

	if answer == "Y" {
		prints := 0
		for true {
			if prints < 5 {
				fmt.Print("Penus  ")
				prints++
			} else {
				fmt.Println("Penus  ")
				prints = 0
			}
		}
	}
}
